gluecode.c

```diff
@@ -0,0 +1,274 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+/*
+ *
+ * Authors: Marius-Cristian Baciu <marius.baciu@stud.acs.upb.ro>
+ *
+ *
+ * Copyright (c) 2020, NEC Europe Ltd., NEC Corporation. All rights reserved.
```

Comment:
```suggestion
 * Copyright (c) 2020, University Politehnica of Bucharest. All rights reserved.
```
This should be like this, as the work done is "from UPB".

Reply:
Done